function-support.html

```diff
@@ -980,8 +1070,9 @@ <h3 id="spacing">Spacing</h3>
     </table>
 	<table>
 		<tr>
-			<td>Notes:</td><td>{<em>distance</em>} will accept any of the <a href="#units">KaTeX units</a>.<br>
-			<code>\mkern</code> and <code>\mskip</code> will not work in text mode and both will write a console warning for any unit except <em>mu</em>.</td>
+			<td style="vertical-align: top;">Notes:</td><td><p>{<em>distance</em>} will accept any of the <a href="#units">KaTeX units</a>.</p>

```

Comment:
This would look better (top aligned) if the first `<p>` gets a `style="margin_top: 0"`, or an analogous CSS rule to remove the top margin from the first `<p>` within a `<td>`.

Reply:
Yep. I'll change it.